.github/workflows/links.yml

```diff
@@ -0,0 +1,25 @@
+name: Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
```

Comment:
Looks like v2 was recently released; can you take a look at https://github.com/lycheeverse/lychee-action/releases/tag/v2.0.0 to see if you can easily update to it?